docs/Contributing/API-for-contributors.md

```diff
@@ -1746,4 +1748,31 @@ If an installer with the provided parameters is found, the installer is returned
 This error occurs if an installer with the provided parameters doesn't exist.
 
 
+### Check if an installer exists 
+
+Checks if a pre-built fleet-osquery installer with the given parameters exists.
+
+`HEAD /api/_version_/fleet/download_installer/{enroll_secret}/{kind}`
+
+#### Parameters
+
+| Name          | Type    | In    | Description                                                        |
+| ------------- | ------- | ----- | ------------------------------------------------------------------ |
+| enroll_secret | string  | path  | The global enroll secret.                                          |
```

Comment:
While we're at it, could we change this to be `query`? (to be consistent)

Reply:
Actually, I think that's already the case and the docs are outdated?

Reply:
thank you! this was a leftover from the previous implementation, fixed.